samples/bookinfo/README.md

```diff
@@ -5,9 +5,11 @@ See <https://istio.io/docs/examples/bookinfo/>
 ## Build docker images without pushing
 
 ```bash
-src/build-services.sh <version>
+src/build-services.sh <version> <prefix>
 ```
 
+Where `<version>` is the tag and `<prefix>` is the docker registry to tag the images.
```

Comment:
Could you include an example of the values to help the reader know which is which?